<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LvjingUIClass</class>
 <widget class="QWidget" name="LvjingUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="lvjing4Btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>165</y>
     <width>47</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/beauty/lj_fennen_tender);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvjing6Btn">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>165</y>
     <width>47</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvjing1Btn">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>100</y>
     <width>47</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/beauty/lj_qingxi_clear);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvjing7Btn">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>165</y>
     <width>47</width>
     <height>47</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvjing2Btn">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>100</y>
     <width>47</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/beauty/lj_qingxin_fresh);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvjingYuanhuaBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>47</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/beauty/beauty_original);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvjing5Btn">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>165</y>
     <width>47</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/beauty/lj_nuanhe_warm);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>80</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>滤镜</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvjing3Btn">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>100</y>
     <width>47</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/beauty/lj_jiari_holiday);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../joy.qrc"/>
 </resources>
 <connections/>
</ui>
